File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        std::string myY;
        std::string myX = std::to_string(x);
        for (int i = myX.length() - 1; i >= 0; --i) {
            myY = myY+ myX[i];
        }
        if (myY == myX) {
            return true;
        } else {
            return false;
        }
    }
};

File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    var myY = '';
    var myX = x.toString();
    
    for (var i = myX.length - 1; i >= 0; --i) {
        myY = myY + myX[i];
    }
    
    return myY === myX;
};